index.html

```diff
@@ -74,6 +74,11 @@ <h2 class="text-primary">Our Vision</h2>
         </div>
       </div>
     </section>
+    <!-- Buttons  -->
+    <div class="but-section">
+      <button class="btn-primary">Join the Community</button>
```

Comment:
Can you add links to the different locations?

Reply:
Yes ..But I was not given any links

Reply:
I would suggest using https://github.com/chryz-hub/chryz-hub.github.io/issues/new?assignees=&labels=invite+me+to+the+organization&template=invitation.md&title=Please+invite+me+to+the+GitHub+Community+Organization for the Join the community

Reply:
Okay